src/lib/public/sitemap/sitemap.ts

```diff
@@ -0,0 +1,9 @@
+export interface SiteMapEntry {
+  id: string;
+  title: string;
+  link: string;
+}
+
+export type SiteMap = {
```

Comment:
Run `npm run all` to automatically resolve any formatting errors.

Reply:
Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/1069/commits/a51a7c94da5aef6b80cdedc39e86ef9147876a12>.